frontend/package.json

```diff
@@ -31,7 +31,7 @@
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
     "typescript": "3.7.3",
-    "yup": "0.28.2"
+    "yup": "0.28.3"
```

Comment:
There is a bug in 28.2 that prevents regexs working properly with custom error messages, fixed in 28.3